sycl/include/CL/sycl/types.hpp

```diff
@@ -873,18 +873,14 @@ template <typename Type, int NumElements> class vec {
   // Special proxies as specialization is not allowed in class scope.
   void setValue(int Index, const DataT &Value) {
     if (NumElements == 1) {
-      setValue(Index, Value, (int)0);
+      setValue(Index, Value, 0);
```

Comment:
How about rewrite using ternary operator like in `getValue` method? Or just elide braces.

Reply:
Ternary operator will not help to avoid type casts.
Removed braces.